### DISCUSSION TITLE
icon sources

### DISCUSSION THREAD

**Discussion by quetzalsly** [mearashadowfax/ScrewFast#77] [Q&A] [answered]
Hey, where did you get the icon sources?
I really need:

discord,
youtube,
instagram,
facebook,
x

Thanks!

**Comment by mearashadowfax** [comment, marked as answer]
> Hey, where did you get the icon sources?

Hi @quetzalsly! From [Simple Icons](https://simpleicons.org/)

**Comment by quetzalsly** [comment]
Thanks, the icons there do look great but I cant tell if they look uneven or not?
![icons_test](https://github.com/mearashadowfax/ScrewFast/assets/38719748/2d4227a7-a34e-40b8-8b52-f84f4f252129)
I just copy pasted the other icons code and only changed the paths.
Should I be trying to find other icons that are all the same dimensions?

**Reply by mearashadowfax** [reply]
I think the icons are looking consistent overall. You can fine-tune the height and width properties for any icons you think are slightly off, right now it's `h-4` and `w-4`.
![Group 121](https://github.com/mearashadowfax/ScrewFast/assets/125820963/ffa76f57-a0d2-41bf-8e93-edc21fefdf64)

**Comment by quetzalsly** [comment]
I tried h-5 w-5 and its better but still not perfect and h-6 w-6 is too big. Damn this is annoying!
I just want the bottom of the icons to align. I have a feeling its because these icons are not from the same set. I am a bit of a perfectionist about these things and slightly unaligned icons really upset me xD

**Reply by mearashadowfax** [reply]
Ugh, I feel you! Trying to get icons to align perfectly can be a real pain. Discord, YouTube, and some other icons are naturally rectangular because their logos are wider than they are tall. Square icons work better for logos that are more balanced shapes. You can try playing around with some padding or margins or find some other icon set.
![image](https://github.com/mearashadowfax/ScrewFast/assets/125820963/e1e8643a-9530-473b-b982-e35f15357da6)
